### DISCUSSION TITLE
For Chrome this upgrade, whether the format is correct

### DISCUSSION THREAD

**Discussion by ddlzr** [Homebrew/discussions#477] [Casks]
![image](https://user-images.githubusercontent.com/22073788/103981016-e8b76780-51bb-11eb-92fd-4c00998d6ae6.png)

**Comment by SMillerDev** [comment]
It's up to Google to number the chrome versions.

**Reply by ddlzr** [reply]
From  the  appcast "https://omahaproxy.appspot.com/history?os=mac;channel=stable", the version number is 87.0.4280.88, not 87.0.4280.88,4280.88. Is our version number incorrectly duplicated ?
![image](https://user-images.githubusercontent.com/22073788/103987902-25895b80-51c8-11eb-9186-8139203a0348.png)

**Reply by SMillerDev** [reply]
@Homebrew/cask can probably answer this.

**Comment by miccal** [comment]
It is a consequence of https://github.com/Homebrew/homebrew-cask/pull/95920 and the automatic update system `brew bump-unversioned-casks` that is being rolled out.
